src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -218,6 +219,14 @@ public bool IsSelectedVersionInstalled
             }
         }
 
+        public bool IsNotAvailableForInstall
```

Comment:
Generally, properties and methods should have positive names, to avoid confusion over double negatives. I realize this is to use the `NotNullOrTrueToBooleanConverter`.  Maybe it's not worthwhile, but would a new converter be useful?

Honestly, by the name alone, I really struggle to guess what `NotNullOrTrueToBooleanConverter` is expected to do. It sounds like it returns `value == null || value == true`. Therefore, if your property was `IsInstalledEnabled`, or `IsInstalledButtonEnabled`, then you don't need a converter?

Reply:
Thank you for the change. It's actually easy to understand now.